services/ux-backend/main.go

```diff
@@ -63,18 +68,25 @@ func main() {
 		os.Exit(-1)
 	}
 
+	cl, err := newClient()
+	if err != nil {
+		klog.Exitf("failed to create client: %v", err)
+	}
+
+	ns := util.GetPodNamespace()
```

Comment:
This contradicts other changes made in some other files. 
Sometimes you assume the secret is in the pod namespace and sometimes you assume it is in the storage cluster namespace (see prev review comment), we need to be consistent

Reply:
ack, using the operator-namespace in storageclient.go